lib/geometry/polygon.nit

```diff
@@ -151,7 +139,38 @@ class ConvexPolygon
 		return projection
 	end
 
-	# Is this polygon convex ?
+	# Remove  `p` from the vertices, keeping at least 3 vertices
+	fun delete_vertex(p: Point[Float]) do if points.length > 3 then points.remove(p)
```

Comment:
By definition a polygon can't have less than 3 points : "a polygon /ˈpɒlɪɡɒn/ is a plane figure that is bounded by a finite chain of straight line segments closing in a loop to form a closed chain or circuit. "